src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -15,6 +15,7 @@ namespace System.Text.RegularExpressions
     /// <summary>
     /// RegexCompiler translates a block of RegexCode to MSIL, and creates a subclass of the RegexRunner type.
     /// </summary>
+    [RequiresDynamicCode("The native code for Regex compilation might not be available at runtime.")]
```

Comment:
"The native code for Regex compilation might not be available at runtime." message does not make sense to me. 

Should this rather be something like "Compiling a RegEx requires dynamic code." ?